WhatsNew.Cli/WhatsNew.Cli.csproj

```
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <Nullable>enable</Nullable>
+	  <ImplicitUsings>enable</ImplicitUsings>
```

nit: Indentation wasn't correct

```suggestion
    <ImplicitUsings>enable</ImplicitUsings>
```